base/applications/notepad/main.c

```diff
@@ -587,7 +591,7 @@ int WINAPI _tWinMain(HINSTANCE hInstance, HINSTANCE prev, LPTSTR cmdline, int sh
     wndclass.lpfnWndProc = NOTEPAD_WndProc;
     wndclass.hInstance = Globals.hInstance;
     wndclass.hIcon = LoadIcon(hInstance, MAKEINTRESOURCE(IDI_NPICON));
-    wndclass.hCursor = LoadCursor(0, IDC_ARROW);
+    wndclass.hCursor = LoadCursor(NULL, (GetSystemMetrics(SM_PENWINDOWS) ? IDC_ARROW : IDC_IBEAM));
```

Comment:
I'm not sure this is necessary. Because the cursor should change to beam automatically when you hover over the edit box.

Reply:
Reverted.